accounts/models.py

```diff
@@ -26,6 +26,7 @@ def is_teacher(self):
 class MyUser(AbstractUser):
     role = models.CharField(max_length=100, choices=USER_ROLE_CHOICES, default=STUDENT)
     email = models.EmailField("email address", blank=True, unique=True)
+    nickname = models.CharField("nickname", max_length=30, blank=True)
```

Comment:
You added the nickname field twice here.

Reply:
The other one is UserConsent, which I should not add